Todo/JS/TodoServerSide/TodoServer_Communication/Controllers/TodoController.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Net.Http;
+using System.Net;
+using Microsoft.AspNetCore.Mvc;
+using TodoServer_Commons;
+using System.Collections.Generic;
+using TodoServer_DL;
+namespace ServerCommunication.Controllers
+{
+    [ApiController]
+    [Route("api/Todo")]
+    public class TodoController : ControllerBase
+    {
+        private static IDB _currentDB = new RamDB();
+
+        public IEnumerable<TodoTask> GetAllTasks()
+        {
+            return _currentDB.GetTasks();
+        }
+
+        [HttpPut]

```

Comment:
its better if you use a specific route for add/edit/delete operations in case you need to add more operations in the future